jbrowse/src/org/labkey/jbrowse/view/browser.jsp

```diff
@@ -92,8 +92,8 @@
                     var config = {
                         containerID: "GenomeBrowser",
                         baseUrl: "<%=h(base)%>",
```

Comment:
This should be q() as well, though the base URL is unlikely to have quotes in it so very low risk.